ui/src/app/customizations/tech-extensions/tech-extensions-list.component.html

```diff
@@ -37,9 +37,9 @@ <h1>Technical Extensions</h1>
               <div class="list-pf-description" text-overflow-pf>{{ item.description }}</div>
             </div>
             <div class="list-pf-additional-content">
-              <div *ngIf="item.integrations">
-                <strong>Used by {{ item.integrations }}</strong>
-              </div>
+              <span>
+                Used by <strong>{{ item.uses || 0 }}</strong> {{ item.uses === 1 ? 'integration' : 'integrations' }}
```

Comment:
This kind of bindings are better managed through Angular's [I18nPluralPipe](https://angular.io/api/common/I18nPluralPipe) pipe, so can automate values such as `no integrations`, `one integration`, `154,456 integrations`...

The caveat is that the controller gets more verbose, but at least the template gets cleaner and less operations are made when interpolating data. I'd add this field in your component controller

```
itemUseMapping: {[valueComparator: string]: string
} = { '=0': 'No integrations', '=1': 'one integration.', 'other': '# integrations' };
```

And then in your template:

``` 
Used by <strong>{{ (item.uses || 0) | i18nPlural: itemUseMapping }}</strong>
``` 

and if you don't feel like tweaking your PR, just go ahead! LGTM LOL

Reply:
There is also I'd like to point out: We should never "shortcircuit" bindings with fallback data in the template (I mean that `item.uses || 0` thingie). Otherwise we'll have to do this everytime we need to inspect and print the value of `item.uses` elsewhere in the code.

I believe the right place for doing this is the service handler that binds the Http response into the `item` model, whatever it is.

Reply:
Awesome, tx!  I'll make those changes, should have an update in a few.